src/config/index.ts

```diff
@@ -18,6 +18,10 @@ const config = {
         redirectUri: env.get('SPIKE_REDIRECT_URI').required().asString(),
         grantType: env.get('SPIKE_GRANT_TYPE').required().asString(),
         responseType: env.get('RESPONSE_TYPE').default('code').required().asString(),
+        rejectUnauthorized: env.get('NODE_TLS_REJECT_UNAUTHORIZED').required().asString(),
```

Comment:
this isn't needed 
config has no use for reject unauthorized
it'll be used anyways if it's configured as a env variable